engines/plumbers/metaengine.cpp

```diff
@@ -40,7 +43,17 @@ class PlumbersMetaEngine : public AdvancedMetaEngine {
 };
 
 Common::Error PlumbersMetaEngine::createInstance(OSystem *syst, Engine **engine, const ADGameDescription *desc) const {
-	*engine = new Plumbers::PlumbersGame(syst, desc);
+	if (desc->platform == Common::kPlatform3DO) {
+#ifdef USE_RGB_COLOR
+		*engine = new Plumbers::PlumbersGame3DO(syst, desc);
+#else
+		// Should never happen. So no need to translate
+		GUI::MessageDialog dialog(_("3DO Plumbers requires RGB support."));
```

Comment:
Please add `// I18N: Plumbers is the title of the game`

This is the comment that the translators will see on Weblate.

Reply:
Done